test/inspector/inspector-helper.js

```diff
@@ -535,6 +535,48 @@ exports.startNodeForInspectorTest = function(callback,
   process.on('exit', handler);
   process.on('uncaughtException', handler);
   process.on('SIGINT', handler);
+
+  return child;
+};
+
+exports.startNodeAndDebuggerViaSignal = function(callback, scriptContents) {
+  const args = ['-e', `${scriptContents}\nconsole.error('started');`];
```

Comment:
Might want to use `process._rawDebug` instead of `console.error` because the latter will trigger an async event.